packages/chart/src/FigureChartModel.ts

```diff
@@ -265,6 +265,12 @@ class FigureChartModel extends ChartModel {
         ? this.dh.plot.DownsampleOptions.DISABLE
         : this.dh.plot.DownsampleOptions.DEFAULT
     );
+
+    if (this.figure.errors.length > 0) {
+      // We don't have a toast or anything we show for errors; just log the error for now
```

Comment:
I think this will fail in DHE because `getErrors` is not marked as `@JsProperty` there, but it's added in the core PR this relies on. Either need a ticket/PR adding the decorator to the DHE JS API or optional chaining here

Also, you can remove the comment

Reply:
Yes, I have a branch already for that. I've created a ticket: https://deephaven.atlassian.net/browse/DH-15932